### ISSUE TITLE
Ready for 3.3.0 PyPI/GitHub release

### ISSUE THREAD

**Issue by Tieqiong** [diffpy/diffpy.structure#117]
### Release checklist for GitHub contributors

- [x] All PRs/issues attached to the release are merged.
- [x] All the badges on the README are passing.
- [x] License information is verified as correct. If you are unsure, please comment below.
- [x] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
  missing), tutorials, and other human written text is up-to-date with any changes in the code.
- [x] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) are updated and
  tested
- [x] Successfully run any tutorial examples or do functional testing in some other way.
- [x] Grammar and writing quality have been checked (no typos).

Please mention @sbillinge when you are ready for release. Include any additional comments necessary, such as
version information and details about the pre-release.

@sbillinge ready for release!

**Comment by sbillinge** [comment]
OK, I did the 3.3.0rc0 release.  Please check and get back t me

**Comment by Tieqiong** [comment]
Yes I can see it on pypi. Docs with new version is also released.

**Comment by sbillinge** [comment]
ok @Tieqiong I made the full 3.3.0 release to pypi.  Please go ahead with the conda-forge workflow.

**Comment by bobleesj** [comment]
@Tieqiong re-visiting here. 

We merged v3.3.0 (supporting py313) on conda-forge but we merged _accidentally_ even though the CI wasn't running and building failed - https://github.com/conda-forge/diffpy.structure-feedstock/pull/26

I suggest that we release v3.3.1 as soon as possible. I've also shared a link under `genomics-and-release` channel under slack for our current group's plan.

**Comment by sbillinge** [comment]
This can be closed after testing an install and run from conda-forge

**Comment by bobleesj** [comment]
````
conda create -n diffpy.structure_env diffpy.structure
conda activate diffpy.structure_env 
```

Locally tested and we can close it  - @sbillinge or @Tieqiong 

```
Locally tested:
=== 137 passed in 0.83s =====
```

**Comment by sbillinge** [comment]
closed as completed!
